### PR TITLE
Add documentation for team switch kit

### PR DESCRIPTION
**Pull request by applenick**
# Add documentation for team switch kit
Screen Shot 2021-02-14 at 12 41 31 PM

See PGMDev/PGM#799 for details. Let me know if there are any suggested changes and I'll adjust asap 👍 

Signed-off-by: applenick <applenick@users.noreply.github.com>

### CODE REVIEW

File: docs/modules/gear/kits.mdx [hunk 1]
```diff
@@ -291,6 +291,71 @@ By default health or hunger values are only increased, not decreased. For exampl
 </kit>
 ```
 
+### Team Switch Kit
+
+Allow for players who recieve this kit to be forced onto a different team.
```
Comment (Indicardo) [inline docs/modules/gear/kits.mdx:296]: receive

**Review by Indicardo — CHANGES_REQUESTED** [review]
Once this typo has been fixed it will be good to go :+1:

File: docs/modules/gear/kits.mdx (hunk 1 above)
Comment (applenick) [reply docs/modules/gear/kits.mdx:296]: Oops my bad

Comment (applenick) [reply docs/modules/gear/kits.mdx:296]: Will fix asap

**Comment by Indicardo** [comment]
Thanks for contributing :+1:

**Pull request merged**